packages/gafl-webapp-service/src/pages/renewals/renewal-inactive/__tests__/route.spec.js

```diff
@@ -1,45 +1,236 @@
+                  endDate: '',
+                  reason: authReason
+                }
+              })
+            }
+          }
+        }),
+        i18n: {
+          getCatalog: () => messages
+        },
+        locale: 'en-gb'
+      })
+
+      it('body message is as expected', async () => {
+        const mockRequest = getMockRequest(authReason)
+        moment.mockImplementation(
+          getMomentMockImpl({
+            format: () => '12th Never'
+          })
+        )
+        const { bodyMessage } = await getData(mockRequest)
+        expect(bodyMessage).toEqual(expectedBodyString)
+      })
+
+      it('title is as expected', async () => {
+        const mockRequest = getMockRequest(authReason, getMessages(mssgOverrides))
+        moment.mockImplementation(getMomentMockImpl())
+        const { title } = await getData(mockRequest)
+        expect(title).toEqual(expectedTitle)
+      })
+    })
+
+    const mockStatusCacheGet = jest.fn()
+
+    const mockRequest = {
+      cache: () => ({
+        helpers: {
+          status: {
+            getCurrentPermission: mockStatusCacheGet
+          }
         }
-      }
-    }),
-    i18n: {
-      getCatalog: jest.fn(() => englishTranslations)
+      }),
+      i18n: {
+        getCatalog: jest.fn(() => englishTranslations)
+      },
+      locale: 'en-gb'
     }
-  }
-  describe('getData', () => {
-    it('should return the reason the renewal is inactive', async () => {
+
+    it.each([
+      [RENEWAL_ERROR_REASON.NOT_DUE, 'not-due'],
+      [RENEWAL_ERROR_REASON.EXPIRED, 'expired'],
+      [RENEWAL_ERROR_REASON.NOT_ANNUAL, 'not-annual']
+    ])('should return the reason', async (reason, expected) => {
+      mockStatusCacheGet.mockImplementationOnce(() => ({ authentication: { reason: reason } }))
+      moment.mockImplementation(getMomentMockImpl())
+      const result = await getData(mockRequest)
+      expect(result.reason).toBe(expected)
+    })
+
+    it('should return the reason codes', async () => {
       mockStatusCacheGet.mockImplementationOnce(() => ({ authentication: { reason: RENEWAL_ERROR_REASON.NOT_DUE } }))
+      moment.mockImplementation(getMomentMockImpl())
       const result = await getData(mockRequest)
-      expect(result.reason).toBe(RENEWAL_ERROR_REASON.NOT_DUE)
+      expect(result.reasonCodes).toBe(RENEWAL_ERROR_REASON)
     })
 
     it('should return the new transation uri', async () => {
       mockStatusCacheGet.mockImplementationOnce(() => ({ authentication: {} }))
+      moment.mockImplementation(getMomentMockImpl())
       const result = await getData(mockRequest)
       expect(result.uri.new).toBe(NEW_TRANSACTION.uri)
     })
 
-    it('should return the body message', async () => {
+    it.each([
+      ['2020-12-13T23:59:59Z', RENEWAL_ERROR_REASON.NOT_DUE, 'ABV123'],
+      ['2020-01-13T23:59:59Z', RENEWAL_ERROR_REASON.NOT_DUE, 'LKJ234'],
+      ['2020-10-13T23:59:59Z', RENEWAL_ERROR_REASON.NOT_DUE, 'MHG356']
```

Comment:
for every one of these, `RENEWAL_ERROR_REASON.NOT_DUE` is passed as a parameter. Given it's the same every time, it's really not a parameter. Should it vary? If not, pass it as a static value.